app/Resources/views/article.html.twig

```diff
@@ -10,7 +10,7 @@
 
     {{ parent() }}
 
-    {% block google_scholar %}
+    {% if google_scholar_metadata ?? false %}
```

Comment:
Could actually move to `article-text.html.twig` and don't then need the default.

Reply:
Not completely related, but the default itself is good as long as the infrastructure change is deployed first